src/wombats/constants.clj

```diff
@@ -6,3 +6,28 @@
 (defonce github-scopes "user:email")
 (defonce github-user-profile-url "https://api.github.com/user")
 (defonce github-repo-api-base "https://api.github.com/repos/")
+
+;; Error Codes
+;; Format: a-bb-ccc
+;; a: File type: 0 handlers
+;;               1 dao
+;; b: File number: Increments by one for each file
+;; c: Error number: Increments by one for each error
+(def errors
```

Comment:
How did you come up with this error format? What if an error exists in multiple files?